src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        int totalSupply = 0;
+        int totalBuy = 0;
+
+        try {
+            List<String> rows = Files.readAllLines(file.toPath());
+            for (String row : rows) {
+                String[] parts = row.split(",");
+                String operation = parts[0];
+                int value = Integer.parseInt(parts[1]);
```

Comment:
It's a good practice to handle `NumberFormatException` when parsing integers from strings. This will make your code more robust against malformed input data.